axis_backend/apps/kpis/views/kpi_viewset.py

```
@@ -0,0 +1,109 @@
+    - Toggle visibility
+    """
+
+    queryset = KPI.objects.all()
+    permission_classes = [IsAuthenticated]
+    service_class = KPIService
+    list_serializer_class = KPIListSerializer
+    detail_serializer_class = KPIDetailSerializer
+    create_serializer_class = KPICreateSerializer
+    update_serializer_class = KPIUpdateSerializer
+
+    def create(self, request, *args, **kwargs):
```

Ruff (0.14.5)

Missing type annotation for `*args`

(ANN002)